ereuse_devicehub/resources/lot/schemas.py

```diff
@@ -16,6 +16,7 @@ class Lot(Thing):
     description = SanitizedStr(description=m.Lot.description.comment)
     closed = f.Boolean(missing=False, description=m.Lot.closed.comment)
     devices = NestedOn(s_device.Device, many=True, dump_only=True)
+    devices = NestedOn(s_device.Device, many=True, dump_only=True)
```

Comment:
@JNadeu  Why is the device field replicated? Not sure if this is the main issue but I imagine this could cause problems

Reply:
My error. I didn't get a good look at it before the commit. It has nothing to do with the issue.